testsuite/forge/src/backend/k8s/chaos.rs

```diff
@@ -32,6 +32,18 @@ macro_rules! NETWORK_LOSS_CHAOS_TEMPLATE {
     };
 }
 
+macro_rules! NETEM_CHAOS_TEMPLATE {
```

Comment:
why are these macros?

Also is it worth just defining this entire struct in code instead of a yaml file?

You can create an struct or hashmap and serialize it after modifying it?

I really hate the helm level of indirection

Reply:
I think this was originally done to enable templating and replace certain placeholders with values from a config struct. I don't see how to do it entirely with a well-defined struct.